src/borg/cache.py

```diff
@@ -180,6 +182,7 @@ def __init__(self, repository, key, manifest, path=None, sync=True, do_files=Fal
         self.timestamp = None
         self.lock = None
         self.txn_active = False
+        self.txn_set = ['config']
```

Comment:
nitpick: naming a list a set is a bit strange. also, I usually avoid having a data type in the name.

txn_files maybe?